holoviews/core/util.py

```diff
@@ -803,15 +803,23 @@ def dimensionless_contents(streams, kdims):
     with any of the key dimensions.
     """
     names = stream_parameters(streams)
-    kdim_names = [kdim.name for kdim in kdims]
-    return [name for name in names if name not in kdim_names]
+    return [name for name in names if name not in kdims]
+
+
+def streamless_dimensions(streams, kdims):
+    """
```

Comment:
I feel we need a better word for ``streamless_dimensions``. These used to be simply 'key dimensions' before streams came along!

In other words, I would like a way to define these things in terms of what they are instead of what they are not. I can't say I have any great suggestions right now 'widget_dimensions', 'requested_dimensions', 'unbound_dimensions'? Got any ideas?